### PR TITLE
Ofmcc 1025 view organization facility

### PR DESCRIPTION
**Pull request by jgstorey**

### CODE REVIEW

File: frontend/src/views/ManageUsersView.vue [hunk 1]
```diff
@@ -34,9 +34,7 @@
               </AppButton>
             </template>
             <template v-slot:item.actions="{ item }">
-              <AppButton @click.stop="toggleDialog(item)" variant="text">
-                edit
-              </AppButton>
+              <AppButton @click.stop="toggleDialog(item)" variant="text">edit</AppButton>
```
Comment (weskubo-cgi) [inline frontend/src/views/ManageUsersView.vue:37]: File should be under account-mgmt.

File: frontend/src/views/ManageUsersView.vue [hunk 2]
```diff
@@ -67,27 +65,21 @@
...
     </v-row>
-    <ManageUserDialog
-      class="pa-0"
-      :show="showManageUserDialog"
-      :updatingUser="userToUpdate"
-      @close="toggleDialog"
-      @close-refresh="closeDialogAndRefresh"
-      @update-success-event="updateSuccessEvent" />
-    <AppButton id="back-account-mgn-button" class="mt-2" :primary="false" size="large" width="400px" :to="{ name: 'account-mgmt' }">&larr; Back to Account Management</AppButton>
+    <ManageUserDialog class="pa-0" :show="showManageUserDialog" :updatingUser="userToUpdate" @close="toggleDialog" @close-refresh="closeDialogAndRefresh" @update-success-event="updateSuccessEvent" />
```
Comment (weskubo-cgi) [inline frontend/src/views/ManageUsersView.vue:80]: Do we need this class? If so can it be moved into ManageUserDialog.vue?

**Review by weskubo-cgi — CHANGES_REQUESTED** [review]

File: frontend/src/views/ManageUsersView.vue (hunk 1 above)
Comment (jgstorey) [reply frontend/src/views/ManageUsersView.vue:37]: Fixed

File: frontend/src/views/ManageUsersView.vue (hunk 2 above)
Comment (jgstorey) [reply frontend/src/views/ManageUsersView.vue]: Removed, not necessary.

**Review by weskubo-cgi — APPROVED** [review]

**Pull request merged**